examples/api/test/widgets/shortcuts/shortcuts.0_test.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/widgets/shortcuts/shortcuts.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('ShortcutsExampleApp', () {
+    testWidgets('displays correct labels', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const example.ShortcutsExampleApp(),
+      );
+
+      expect(find.text('Shortcuts Sample'), findsOneWidget);
+      expect(
+        find.text('Add to the counter by pressing the up arrow key'),
+        findsOneWidget,
+      );
+      expect(
+        find.text('Subtract from the counter by pressing the down arrow key'),
+        findsOneWidget,
+      );
+      expect(find.text('count: 0'), findsOneWidget);
+    });
+
+    testWidgets(
+      'updates counter on arrowUp and arrowDown press',
+      (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('updates counter on arrowUp and arrowDown press', (WidgetTester tester) async {
```

Reply:
The length of this line is greater than 80 chars, so formatter separates it into two parts.  @TahaTesser is there any suggestion to ignore formatter in such cases?

Reply:
If the single `group`, this line would align closer to start and if it still cross 80 chars, there are only couple of tests here.

Reply:
When adding group, the extra indent is needed  for this test title but we just have single group we can remove the group.